Cython/Compiler/Parsing.py

```diff
@@ -3782,6 +3782,7 @@ def p_cpp_class_definition(s, pos,  ctx):
         s.next()
         s.expect('NEWLINE')
         s.expect_indent()
+        doc = p_doc_string(s)
```

Comment:
This doesn't seem to be used. Maybe not assign it at all and add a comment instead that it's optional and useless?

Reply:
You're absolutely right, this will be clearer.